collections/drop_while.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Drop all elements in the given collection until the first element that does not match the given predicate.
+ *
+ * Example:
+ *
+ * ```ts
+ * import { dropWhile } from "./drop_first_while.ts";
+ * import { assertEquals } from "../testing/asserts.ts";
+ *
+ * const numbers = [ 3, 2, 5, 2, 5 ]
+ * const dropWhileNumbers = dropWhile(numbers, i => i !== 2)
+ *
+ * assertEquals(dropWhileNumbers, [ 5, 2, 5 ])
```

Comment:
That examples looks wrong. Shouldn't this be `[ 2, 5, 2, 5 ]`?

Reply:
As i know we should not include predictable number,

Reply:
I don't think I understand what you mean.

`dropWhile` should drop all elements from the start that match the given predicate, until it hits one that doesnt. The one that doesnt should not be dropped, as it does not match the predicate.

Reply:
@LionC fixed